packages/caver-transaction/src/transactionTypes/abstractTransaction.d.ts

```diff
@@ -0,0 +1,68 @@
+/*
+    Copyright 2021 The caver-js Authors
+    This file is part of the caver-js library.
+
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import Account from '../../../caver-account/src'
+import { KlaytnCall } from '../../../caver-rpc/src/klay'
+import { Keyring } from '../../../caver-wallet/src/keyring/keyringFactory'
+import SignatureData from '../../../caver-wallet/src/keyring/signatureData'
+
+export interface CreateTransactionOptions {
+    account?: Account
+    from?: string
+    to?: string
+    value?: string | number
+    input?: string
+    nonce?: string
+    gas?: string | number
+    gasPrice?: string
+    chainId?: string
+    feePayer?: string
```

Comment:
```suggestion
    feePayer?: string
    signatures?: string[] | SignatrueData[]
    feePayerSignatures?: string[] | SignatrueData[]
    feeRatio?: string | number
    humanReadable?: boolean
    codeFormat?: string
```

Reply:
It occurs error `caver.validator.js` lines at 776, 1453... 

How about below?

```
export interface CreateTransactionOptions {
    ...
    signatures?: string[] | string[][] | SignatureData[]
    feePayerSignatures?: string[] | string[][] | SignatureData[]
    ...
}
```

Reply:
Oh right, please define type like what you said.

Reply:
Fixed it. please check 0111a54